integration-web/src/main/java/org/symphonyoss/integration/web/resource/WebHookResource.java

```diff
@@ -201,6 +202,19 @@ public void run() {
   }
 
   /**
+   * Handle {@link NotFoundException} exception.
+   * @param ex Exception object
+   * @return HTTP 404 (Not Found)
+   */
+  @ResponseBody
+  @ExceptionHandler(NotFoundException.class)
+  public ResponseEntity<String> handleNotFound(Exception ex) {
+    String message = ex.getMessage();
+    LOGGER.info(message, ex);
```

Comment:
You might remove the stack trace.

Something like that: LOGGER.info(ex.getMessage());

The variable message is not required as well.